nixnet_examples/programmatic_database_api.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pprint
+import six
+import time
+
+import nixnet
+from nixnet import constants
+from nixnet import types
+from nixnet.system import system
+
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+def main():
+    with system.System() as my_system:
+        print(list(my_system.databases))
+        print(list(my_system.databases.keys()))
+        print(list(my_system.databases.values()))
+        print(list(my_system.databases.items()))
+        print(len(my_system.databases))
+        database_alias = 'custom_database'
+        database_filepath = 'databases/custom_database.dbc'
```

Comment:
Probably want to prefix this with `basename(__file__)` or whatever the call would be so they can run this from other locations